.pre-commit-config.yaml

```diff
@@ -23,3 +23,8 @@ repos:
       - id: "check-toml"
       - id: "check-yaml"
       - id: "check-merge-conflict"
+
+  - repo: "https://github.com/teemtee/tmt"
+    rev: main
+    hooks:
+      - id: tmt-lint
```

Comment:
```suggestion
  - repo: "https://github.com/teemtee/tmt"
    rev: "1.17.0"
    hooks:
      - id: tmt-lint
```

Reply:
Oh cool, I was wondering why it was complaining :))
Thank you, @r0x0d!